tests/unit/amo/server/test_base.js

```diff
@@ -560,6 +560,115 @@ describe(__filename, () => {
       expect(users.currentUserWasLoggedOut).toEqual(true);
       mockUsersApi.verify();
     });
+
+    it('sets a X-Accel-Expires header if request is safe & anonymous and response is sucessful', async () => {
+      const { store, sagaMiddleware } = createStoreAndSagas();
+
+      const response = await testClient({ store, sagaMiddleware }).get(
+        '/en-US/firefox/',
+      );
+      expect(response.headers['X-Accel-Expires'.toLowerCase()]).toEqual('180');
+    });
+
+    it('does not set a X-Accel-Expires header if request contained token cookie', async () => {
+      const { store, sagaMiddleware } = createStoreAndSagas();
+
+      const response = await testClient({ store, sagaMiddleware })
+        .get('/en-US/firefox/')
+        .set('cookie', `${defaultConfig.get('cookieName')}="foo"`);
+      expect(response.headers['X-Accel-Expires'.toLowerCase()]).toBeUndefined();
+    });
+
+    it('does not set a X-Accel-Expires header if request method is not read-only', async () => {
```

Comment:
the terminology you used elsewhere is "safe" so I'd keep that here (instead of read-only)